components/FlyForm.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+
+class FlyForm extends Component {
+  state = {
+    name: '',
+    color: '',
+    size: '',
+    type: '',
```

Comment:
I think the backend had to change it from type to category, but this works for now since we aren't hooked up to the API.

Reply:
Good to know. This is just the initial setup. We can always change it when we determine which properties are correct based on the BE setup of the flies database.

Reply:
Sounds good!

Reply:
Correct, it the attribute is now referred to as category